src/MonoMod.UnitTest/RuntimeDetour/DetourRedoTest.cs

```diff
@@ -4,15 +4,15 @@
 extern alias New;
 
 using Xunit;
-using New::MonoMod.RuntimeDetour;
-using System;
-using System.Reflection;
-using System.Runtime.CompilerServices;
-using MonoMod.Utils;
-using System.Reflection.Emit;
-using System.Text;
-using System.Collections.Generic;
-using System.Collections.Concurrent;
+// using New::MonoMod.RuntimeDetour;
+// using System;
+// using System.Reflection;
+// using System.Runtime.CompilerServices;
+// using MonoMod.Utils;
+// using System.Reflection.Emit;
+// using System.Text;
+// using System.Collections.Generic;
+// using System.Collections.Concurrent;
```

Comment:
This test needs to be rewritten entirely... I'd rather leave it as is until that happens, to serve as something of a reminder.